common/src/main/scala/models/Topic.scala

```diff
@@ -7,6 +7,7 @@ import cats.data.Xor
 
 case class Topic(`type`: TopicType, name: String) {
   override def toString: String = s"${`type`}/$name"
+  def androidToString: String = s"${`type`}//$name"
```

Comment:
[Nitpicking] I think I would rather see the logic duplicated at the time it's needed than putting platform specific logic on a generic model